test/mongo-read-stream.test.ts

```diff
@@ -120,6 +120,73 @@ ava('[MONGO-READ-STREAM] Create stream with hint OK', async (t: Assertions) => {
 	await mongoClient.dropCollection();
 });
 
+ava('[MONGO-READ-STREAM] Create stream with sort OK', async (t: Assertions) => {
+	const mongoClient = new MongoClient(`test-${Date.now()}`, TestItem, TestItem);
+
+	for (let i = 0; i < 20; i += 1) {
+		// alternate value for testing sort on i
+		const value = i % 2 ? i : i + 20;
+		await mongoClient.insertOne({ key: `value-${Math.random()}`, i: value }, 'userId1', false);
+	}
+
+	// sort OK
+	let lastItem: TestItem;
+	let stream = mongoClient.stream({}, 5, undefined, undefined, { i: 1 });
+	await stream.forEachAsync(async (item: TestItem) => {
+		if (!item) t.fail('missing item');
+		if (lastItem && lastItem.i > item.i) t.fail('bad sorting');
+
+		lastItem = item;
+		await waitFor(2); // add some time to simulate long process
+	});
+	t.falsy(stream.query.$and);
+
+	lastItem = null;
+	stream = mongoClient.streamWithType({}, TestItem, 5, undefined, undefined, { i: -1 });
```

Comment:
if you keep the sort, add a test with composed sorts

Reply:
done!